website/package.json

```diff
@@ -21,7 +21,7 @@
     "dependencies": {
         "@astrojs/node": "^8.3.4",
         "@auth/core": "^0.37.4",
-        "@genspectrum/dashboard-components": "^0.10.0",
+        "@genspectrum/dashboard-components": "0.10.1",
```

Comment:
```suggestion
        "@genspectrum/dashboard-components": "^0.10.1",
```

Reply:
Nice catch. It was a leftover from the local use.